src/common/template.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import fs from "fs-extra";
+import Handlebars from "handlebars";
```

Comment:
Currently the class uses the default/global Handlebars instance. I thought that yesterday we had decided on using a custom instance? (So that we don't have accidental collisions if someone is using the global instance elsewhere.)